src/test/java/de/qabel/desktop/hockeyapp/VersionClientTest.java

```diff
@@ -0,0 +1,162 @@
+package de.qabel.desktop.hockeyapp;
+
+import de.qabel.core.accounting.CloseableHttpClientStub;
+import de.qabel.core.accounting.CloseableHttpResponseStub;
+import org.apache.http.NameValuePair;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONException;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class VersionClientTest {
+
+
+    private static final String VERSION_1_0 = "1.0";
+    private static final String VERSION_1_1 = "1.1";
+
+    private CloseableHttpClientStub httpClient = new CloseableHttpClientStub();
+    private HockeyAppRequestBuilder requestBuilder = new HockeyAppRequestBuilder(VERSION_1_0, httpClient);
+    private VersionClient client = new VersionClient(requestBuilder);
+
+    @Test
+    public void validFindVersion() throws IOException, VersionNotFoundException {
+        loadFakeVersions(200, true);
```

Comment:
afaik loadFakeVersions is always called with 200 as the first parameter, so you can just remove it as a parameter